app/views/audits/audits/_content_items.html.erb

```diff
@@ -1,14 +1,14 @@
 <%= render 'audits/common/counter', count: content_items.total_count  %>
 
-<table class="table table-bordered table-hover">
+<table class="table table-bordered table-hover filter-list">
```

Comment:
[Minor] Could potentially use a data-test attribute to identify elements in the tests rather than changing the class

Reply:
Ah I didn't know about `data-test` as a convention. I prefer that so I'll make some changes. Thanks!